src/main/java/duke/core/Parser.java

```diff
@@ -56,5 +57,39 @@ public Parser(String userInput) {
         }
     }
 
+    public String parseDeletePatient() throws DukeException {
+        String formattedInput;
+        String inputToParse = userInput.replace("delete patient", "").trim();
+
+        if (inputToParse.contains("#")) {
+            try {
+                formattedInput = inputToParse.replace("#", "").trim();
+                return formattedInput;
+            } catch(Exception e){
+                throw new DukeException("Please follow the format 'delete patient #<id>'.");
+            }
+        } else {
+            formattedInput = inputToParse;
+            return formattedInput;
+        }
+    }
+
+    public String parseDeleteTask() throws DukeException {
+        String formattedInput;
+        String inputToParse = userInput.replace("delete task", "").trim();
```

Comment:
Good point! I'll go back and do an initial comparison of the input after converting it to lowercase to check for 'delete task/delete patient', then take the original input and parse that for the final format so that the case-sensitivity is still present.